Add intermediate products (#143)
ASFHyP3/hyp3-back-projection: CHANGELOG.md

and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.9.9]

### Changed

> * The `time_series` workflow now includes intermediate products such as the interferograms in the uploaded file.

Reviewer: ```suggestion
* The `time_series` workflow now includes the intermediate interferogram, correlation, and amplitude files in the packaged product.
```
